instrumentation/opentelemetry_oban/mix.exs

```diff
@@ -45,10 +45,10 @@ defmodule OpentelemetryOban.MixProject do
   defp deps do
     [
       {:oban, "~> 2.0"},
-      {:opentelemetry_api, "~> 1.2"},
+      {:opentelemetry_api, "~> 1.4"},
```

Comment:
Is there a need to further restrict the `opentelemetry_api` version requirement as part of this?